content/master/concepts/managed-resources.md

```diff
@@ -839,96 +839,143 @@ Providers may define their own custom `Conditions`.
 {{</hint >}}
 
 
-### Available
-`Reason: Available` indicates the Provider created the managed resource and it's
-ready for use. 
+### Type: Ready
 
-```yaml {copy-lines="none"}
+Condition `Type: Ready` indicates if the external resource is ready to use.
+
+The Condition `Type: Ready` and `Status: True` indicates Provider created
+the external resource and notified Crossplane it's ready to use. Either the
+external API explicitly indicated the resource is ready or Crossplane appears to
+have successfully created the external resource.
+
+{{<hint "important" >}}
+Crossplane doesn't update `Status: True` during a resource update.
+{{< /hint >}}
+
+The Condition `Type: Ready` and `Status: False` indicates the external resource
+isn't available. 
+
+
+#### Available
+{{<hover label="available" line="4">}}Reason: Available{{</hover>}} indicates 
+the Provider created the managed resource and it's ready for use. 
+
+```yaml {copy-lines="none",label="available"}
 Conditions:
   Type:                  Ready
   Status:                True
   Reason:                Available
 ```
-### Creating
 
-`Reason: Creating` indicates the Provider is attempting to create the managed
-resource. 
+#### Creating
 
-```yaml {copy-lines="none"}
+{{<hover label="creating" line="4">}}Reason: Creating{{</hover>}} indicates the 
+Provider is attempting to create the managed resource. 
+
+```yaml {copy-lines="none",label="creating"}
 Conditions:
   Type:                  Ready
   Status:                False
   Reason:                Creating
 ```
 
-### Deleting
-`Reason: Deleting` indicates the Provider is attempting to delete the managed
-resource. 
+#### Deleting
+{{<hover label="deleting" line="4">}}Reason: Deleting{{</hover>}} indicates the 
+Provider is attempting to delete the managed resource. 
 
-```yaml {copy-lines="none"}
+```yaml {copy-lines="none",label="deleting"}
 Conditions:
   Type:                  Ready
   Status:                False
   Reason:                Deleting
 ```
 
-<!-- vale off -->
-### ReconcilePaused
-<!-- vale on -->
-`Reason: ReconcilePaused` indicates the managed resource has a [Pause](#paused)
-annotation 
+#### Unavailable
+{{<hover label="unavailable" line="4">}}Reason: Unavailable{{</hover>}}
+indicates Crossplane expects the managed resource to be available, but the 
+Provider reports the resource is unhealthy.
 
-```yaml {copy-lines="none"}
+```yaml {copy-lines="none",label="unavailable"}
 Conditions:
-  Type:                  Synced
+  Type:                  Ready
   Status:                False
-  Reason:                ReconcilePaused
+  Reason:                Unavailable
 ```
 
+### Type: Synced
+
+Condition `Type: Synced` indicates Crossplane has checked the status of the managed
+resource with the Provider.
```

Comment:
Is there a verb other than "checked" that could work here?

Checked reads as if "syncing" a resource is a read-only operation, but it isn't.

I like "update or determine" below if that's not too unwieldy here. So "Crossplane has updated and determined the status of the managed resource".

Reply:
How about "validated the status of the managed resource..."?

I'm trying to break the Condition and Status into two parts, where the Condition is "we tried to inspect at the MR" and the status is "inspection worked and now update if needed". Because of that I want to avoid "updated" in the first paragraph.

Reply:
> I'm trying to break the Condition and Status into two parts, where the Condition is "we tried to inspect at the MR" and the status is "inspection worked and now update if needed".

Breaking it into two parts doesn't make sense to me. The status condition represents the entire reconcile operation, not only the inspection of the MR.

Maybe writing out what happens in pseudocode would help explain this better:

```python
def sync_resource():
  desired = read_desired_state()
  actual = read_actual_state()
  if actual != desired:
    update_actual_state(desired)

def main():
  while True:
    try:
      sync_resource()
    except SyncException:
     set_condition_synced_false()
    else:
     set_condition_synced_true()
    sleep(60)
```

Most reconcile loops boil down to read desired state, read actual state, update actual state to match desired state. If __all__ of that works we set `Synced: true`. If __any__ of that fails we set `Synced: false`.

Reply:
Got it.

When does a resource move from Ready/True to Synced/True? On the first reconcile after it's ready?

Reply:
> When does a resource move from Ready/True to Synced/True?

The two condition types (Synced and Ready) are independent of each other, so I wouldn't think a resource moving from one to the other.

That said, Synced: True is usually going to happen first, because for Synced to be True the controller just needs to get through one reconcile loop without hitting an error.

Let's say it's the very first reconcile of an MR, and the code managed to observe the external resource, notice it doesn't exist, and create it. Synced would be True because the controller just "synced" the desired state. In this case, by creating the external resource.

Often Ready: True would happen a bit later, because many MR controllers observe the external resource to see whether it appears to be ready (according to the API). So at that point the MR would be Synced: True and Ready: True.

Reply:
Expanding on the pseudocode to include the ready condition:

```python
def sync_resource():
  desired = read_desired_state()
  actual = read_actual_state()
  if actual != desired:
    update_actual_state(desired)
  return actual

def main():
  while True:
    try:
      actual_state = sync_resource()
      if is_ready(actual_state):
        set_condition_ready_true()
      else:
        set_condition_ready_false()
    except SyncException:
     set_condition_synced_false()
    else:
     set_condition_synced_true()
    sleep(60)
```